CONTRIBUTION.md

```diff
@@ -1,94 +1,136 @@
+contributions into the project.  
 
+## New Contributor guide
 
-## Reporting bugs and creating issues
+To get an overview of what the WMCO is and does, please take a look at the [README](README.md) first. In order to
+work with the codebase, you'll need to understand a few basic git concepts. Follow the steps below to set up your
+environment to get started.
 
-If any part of the project has bugs or documentation mistakes, please let us know by opening a
-[Jira issue](https://jira.coreos.com/projects/WINC/summary)
+- [Fork the repository on GitHub.](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+- [Clone the forked repository outside your go path](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+- [Create a topic branch, branched off of master.](https://www.atlassian.com/git/tutorials/using-branches)
+- [Make commits of logical units as you work](https://github.com/git-guides/git-commit)
 
-## Contribution flow
+## Before opening a pull request
 
-This is an outline of what a contributor's workflow looks like:
+Thank you for contributing to the Windows Machine Config Operator!
 
-- Fork the repository on GitHub.
-- Clone the forked repository outside your go path, or refer to the [Kubernetes repo's example](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage).
-- Create a topic branch, branched off of master.
-- Make commits of logical units. A commit should typically add a feature or fix a bug, but never both at the same
-time. Vendor commits should always be separate.
-- A PR should consist of a set of logically connected commits to the issue that makes it easy to review and should not 
-follow your personal development flow.
-- Make sure commit messages are in the proper format (see below).
-- Push changes in a topic branch to a personal fork of the repository.
-- To make sure that your topic branch is in sync with the remote master branch,
-follow a [rebase workflow](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
-- Submit a pull request to openshift/windows-machine-config-operator (see PR workflow below).
-- The PR must receive one `/lgtm` and one `/approve` comments from the maintainers of the project.
+Before opening a pull request, be sure that your changes have been tested, documented, and checked for style. Once
+you think your PR is ready for review, be sure to check that you have
 
-Thanks for contributing!
+- [fetched from upstream](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository#fetching-changes-from-a-remote-repository)
+- [Rebased your changes against your root branch](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+- [Run the tests locally](docs/HACKING.md)
+- linted your changes with ```make lint```
+- ensured error messages are a single line
+- updated relevant documentation if your PR changes user functionality  
```

Comment:
Be consistent with capitalizing/not capitalizing the bullets - pick one and stick with it